### PR TITLE
Publishing github LinkedIn Kafka artifacts to bintray

### PR DESCRIPTION
**Pull request by jonlee2**
To publish a new release, use the following command:
./gradlew -Pversion=<version> uploadArchivesAll

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -177,9 +182,9 @@ subprojects {
               afterEvaluate {
                   pom.artifactId = "${archivesBaseName}"
                   pom.project {
-                      name 'Apache Kafka'
+                      name 'LinkedIn branch of Apache Kafka'
```
Comment (radai-rosenblatt) [inline build.gradle]: you probably want to call this a fork

File: build.gradle [hunk 2]
```diff
@@ -517,7 +522,7 @@ for ( sv in availableScalaVersions ) {
 }
 
 def connectPkgs = ['connect:api', 'connect:runtime', 'connect:transforms', 'connect:json', 'connect:file', 'connect:basic-auth-extension']
-def pkgs = ['clients', 'examples', 'log4j-appender', 'tools', 'streams', 'streams:streams-scala', 'streams:test-utils', 'streams:examples'] + connectPkgs
+def pkgs = ['clients', 'examples', 'log4j-appender', 'tools', 'streams', 'streams:test-utils', 'streams:examples'] + connectPkgs
```
Comment (radai-rosenblatt) [inline build.gradle:525]: if we drop streams-scala, why keep all the other streams* ?

**Comment by radai-rosenblatt** [comment]
you also want to ammend the commit msg to make it clear this is to be called by the CI pipeline and not individual devs

**Review by radai-rosenblatt — APPROVED** [review]

File: build.gradle (hunk 1 above)
Comment (jonlee2) [reply build.gradle]: Done

**Pull request merged**